content/rs/administering/designing-production/networking/port-configurations.md

```diff
@@ -29,6 +29,8 @@ update your firewall with the port for that new database endpoint.
 | 8001 | Used by your application to access the RS [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) | TCP |
 | 8443 | Used for secure (https) access to the management web UI | TCP |
 | 8444, 9080 | Used for nginx \<-\>cnm_http/cm communications on the same host only. Ports are bound to loopback adapter. | TCP |
+| 9081 | Used for crdb_coordinator | TCP |
```

Comment:
@ushachar - can you please approve these ports are new and required for RS deployments- 9081 (- CRDB management) and 8070, 8071 (- metrics exporter management)

Reply:
@RoeyPrat  ^^^^ (another two ports)